types/mathjs/index.d.ts

```diff
@@ -304,7 +307,7 @@ declare namespace mathjs {
 		 * @param x The base
 		 * @param y The exponent
 		 */
-		pow(x: number|BigNumber|Complex|MathArray|Matrix, y: number|BigNumber|Complex): number|BigNumber|Complex|MathArray|Matrix;
+		pow(x: MathType, y: number|BigNumber|Complex): MathType;
```

Comment:
These changes caused a few unit tests to fail. This is because `BigNumber` before was `{}`, allowing nearly any value to be passed into this function. Specifically the test was passing a `Unit`.